integrated_channels/cornerstone/client.py

```diff
@@ -43,24 +43,28 @@ def create_content_metadata(self, serialized_data):
         Create content metadata using the Cornerstone course content API.
         Since Cornerstone is following pull content model we don't need to implement this method
         """
+        return 200, ''
```

Comment:
These methods were never called in the past. I am calling them now and its easier to have them return a fake response code and body. Cornerstone pulls updates via an API call versus us sending updates over. I've refactored some of the way we accomplish that to simplify things and remove inconsistencies. This change helps that along.